include/z64audio.h

```diff
@@ -434,7 +441,7 @@ typedef struct SequenceChannel {
     /* 0x80 */ AdsrSettings adsr;
     /* 0x88 */ NotePool notePool;
     /* 0xC8 */ s8 soundScriptIO[8]; // bridge between sound script and audio lib, "io ports"
-    /* 0xD0 */ u8* unk_D0; // New to MM
+    /* 0xD0 */ u8* sfxState; // New to MM
```

Comment:
Right, I was using `New to MM` all over the audio repo in my WIP in decompiling/documenting since everything was done relative to OoT. I can remove this as I've been removing this in other cases

Reply:
Yeah, I think an overview document on the differences would probably be better than commenting on everything inline if there's a lot of changes.